opendevin/core/config.py

```diff
@@ -185,6 +201,22 @@ def defaults_to_dict(self) -> dict:
                 dict[f.name] = get_field_info(f)
         return dict
 
+    def __str__(self):
+        attr_str = []
+        for f in fields(self):
+            attr_name = f.name
+            attr_value = getattr(self, f.name)
+
+            if attr_name in ['e2b_api_key', 'github_token']:
```

Comment:
Here ssh_password is missing.

Reply:
They're likely newer than this PR. Thanks for hunting them down!